src/codegen/return_value.rs

```diff
@@ -43,13 +43,29 @@ impl ToReturnValue for library::Parameter {
 impl ToReturnValue for analysis::return_value::Info {
     fn to_return_value(&self, env: &Env, is_trampoline: bool) -> String {
         match self.parameter {
-            Some(ref par) => par.to_return_value(env, is_trampoline),
+            Some(ref par) => {
+                let name = par.to_return_value(env, is_trampoline);
```

Comment:
Also, shouldn't it be done inside this method instead?

Reply:
Inside that function is more generic and also used in other contexts than return value of normal functions, that's why I've put it outside